<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuNetworkWidget</class>
 <widget class="QWidget" name="MenuNetworkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>540</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bottomLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>【菜单】退出页面  【红键】保存设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="maskLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子网掩码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="gatewayLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网关：</string>
   </property>
  </widget>
  <widget class="QLabel" name="dnsLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS：</string>
   </property>
  </widget>
  <widget class="KCheckBox" name="dhcpCheck">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自动获取IP</string>
   </property>
  </widget>
  <widget class="KIPEdit" name="ipEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="KIPEdit" name="maskEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="KIPEdit" name="gatewayEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="KIPEdit" name="dnsEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KIPEdit</class>
   <extends>QLineEdit</extends>
   <header>src/osd/widget_ip_edit.h</header>
  </customwidget>
  <customwidget>
   <class>KCheckBox</class>
   <extends>QCheckBox</extends>
   <header>src/osd/widget_check_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
